spec/requests/api/v1/directories_spec.rb

```diff
@@ -17,15 +17,15 @@
           user: Fabricate(:user, confirmed_at: nil, approved: true),
           username: 'local_unconfirmed'
         )
-        local_unconfirmed_account.create_account_stat!
+        local_unconfirmed_account.account_stat.save!
```

Comment:
The code for `account_stat` starts to behave weirdly when `autosave: true` is specified for the `account_secret` association 🤔

Reply:
What issues?

Reply:
The objects of `account_stat` are already saved to the database in places where we expected them to not be saved yet. That's why I had to change `create_account_stat!`, because it started throwing exceptions.